src/utils/Hashing.ts

```diff
@@ -1,12 +1,10 @@
-import CID from 'cids'
-import multihashing from 'multihashing-async'
-import ipfsHashing from 'ipfs-only-hash'
-import { ContentFileHash, EntityContentItemReference, EntityMetadata } from '../types'
+import * as hashing from '@dcl/hashing'
+import { ContentFileHash } from '../types'
 
 export namespace Hashing {
   /**
    * Given a set of files, return a map with their hash
-   * @deprecated use calculateIPFSHashes instead
+   * @deprecated use calculateIPFSHashes instead. This function only exists for compatibility reasons
```

Comment:
shouldn't we remove this method?

Reply:
Yes, I wasn't planning to do it but I will take the chance of the breaking change to do it. On it